layout-webapp/src/main/resources/locale/portlet/SiteNavigation_en.properties

```
@@ -1,6 +1,6 @@
 siteNavigation.button.tooltip.label=Site navigation
 siteNavigation.drawer.title=Site navigation
-siteNavigation.label.btn.createNode=Create a node
+siteNavigation.label.btn.createNode=Add Item
```

Should be managed by PO on crowdin even if it's possible to do with code

```suggestion
siteNavigation.label.btn.createNode=Create a node
```

---

removed! 
cc @srenault-meeds FYI